### ISSUE TITLE
Wrong blueprint names

### ISSUE THREAD

**Issue by felixlinker** [EDCD/coriolis#519]
There are a couple of blueprints named wrong. Here is a table of what has been encountered by me so far:

| Coriolis-Name | Actual Name |
| ---------------- | -------------- |
| `Sensor_Sensor_LongRange` | `Sensor_LongRange` |
| `SurfaceScanner_Expanded` | `Sensor_Expanded` |
| `Sensor_KillWarrantScanner_FastScan` | `Sensor_FastScan` |
| `Sensor_WakeScanner_FastScan` | `Sensor_FastScan` |
| `Sensor_CargoScanner_FastScan` | `Sensor_FastScan` |

**Comment by felixlinker** [comment]
Same for `_WideAngle` and `_LongRange` mods on scanners. They also consume the same materials therefore are the same blueprints.ö

**Comment by felixlinker** [comment]
@willyb321 I'd opt in to remove all the utility-related blueprints and just make the model simpler. Also rename the `_Reinforced`, `_FastScan` and `_Shielded` to `Sensor_Reinforced`, etc. That'd remove a couple of redundant blueprints again.

Only problem is the `uuid` field on the blueprints. Those differ. I only found references to `uuid` in [ModalShoppingList.jsx](https://github.com/EDCD/coriolis/blob/master/src/app/components/ModalShoppingList.jsx) so I guess this has to do with EDEngineer interoparability?

**Comment by willyb321** [comment]
We already have an issue with unified BPs on weapons so I'd like to avoid them if possible

**Comment by felixlinker** [comment]
`HeatSinkLauncher_LightWeight` is called `Misc_LightWeight`.

**Comment by felixlinker** [comment]
In fact, all of Heatsink/ECM/Chaff/Pointdefence mods that are Reinforced/Shielded/Lightweight are called `Misc_Reinforced`/`_Shielded`/`_Lightweight`.

Futhermore, `ChaffLauncher_ChaffCapacity` is called `Misc_ChaffCapacity`, `HeatSinkLauncher_HeatSinkCapacity` is called `Misc_HeatSinkCapacity` and `PointDefence_PointDefenseCapacity` is called `Misc_PointDefenseCapacity`.

**Comment by felixlinker** [comment]
@willyb321 The only problem we have are overcharged multi-cannons which reduce ammo-capacity for overcharged. Two months ago you added a special blueprint for rapid fire cannons but that is not used anywhere and both inara and edshipyard agree on the rapid fire blueprints.

I think, we would be better of removing this redundancy and handle the reduction of ammo capacity someway differently. I see two problems with the approach of redundancy:

1. We have to de-multiplex the fdname keys when importing journal builds
2. We're inconsistent about it and have redundancies where there shouldn't be any

![image](https://user-images.githubusercontent.com/9728715/58748055-b2f4bd80-8473-11e9-9ccb-0f80a4d5ef30.png)

**Comment by felixlinker** [comment]
@willyb321 You can preview changes at https://github.com/EDCD/coriolis-data/tree/fix/misc

**Comment by felixlinker** [comment]
So, for summary. I fixed all the utility blueprint names. These now are imported correctly from journal builds. Beforehand, modules would show up as modified but without a specific blueprint given.

This does not break builds. I checked [this build](/outfit/anaconda?code=A0pyttFolodDsyf5------1717000003030101020005044j-03------00-.Iw18ZXEA.Aw18ZXEA.H4sIAAAAAAAAAxP4x8vAwPCXFUjwV7z5%2F1%2FozL3%2F%2F%2F8L%2FuODixqUMjAIzbBiYPgv9M8MRVRCQB0oKvzPBIsJIv9EsJgg%2Bk8Uiwli%2F1RBosxAgqFDGKgWJCoDYv0X%2F8eDReoFkP%2F%2FPwMUAABBVMt0wwAAAA%3D%3D.EweloBhBmUEY4BYQFMCGBzANikI4SEhQlA%3D%3D&bn=deal%20breaker) on my local dev branch and it has all the utilities modified correctly with the only downside being that the modules now shop up with no specific blueprint applied. But I think this is okay.
